pkg/operator/ceph/object/spec.go

```diff
@@ -166,21 +168,32 @@ func (c *clusterConfig) makeRGWPodSpec(rgwConfig *rgwConfig) (v1.PodTemplateSpec
 	if err != nil {
 		return v1.PodTemplateSpec{}, err
 	}
-	if kmsEnabled {
-		if c.store.Spec.Security.KeyManagementService.IsTokenAuthEnabled() {
-			vaultFileVol, _ := kms.VaultVolumeAndMount(c.store.Spec.Security.KeyManagementService.ConnectionDetails,
-				c.store.Spec.Security.KeyManagementService.TokenSecretName)
-			tmpvolume := v1.Volume{
-				Name: rgwVaultVolumeName,
-				VolumeSource: v1.VolumeSource{
-					EmptyDir: &v1.EmptyDirVolumeSource{},
-				},
-			}
+	s3Enabled, err := c.CheckRGWS3()
+	if err != nil {
+		return v1.PodTemplateSpec{}, err
+	}
+	if kmsEnabled || s3Enabled {
+		tmpvolume := v1.Volume{
```

Comment:
Variables used immediately can be a single letter
```suggestion
		v := v1.Volume{
```